allofplos/gdrive.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import os
+import tarfile
+from zipfile import ZipFile, BadZipFile
+
+import requests
+from tqdm import tqdm
+
+from . import get_corpus_dir
+
+# Variables needed
+zip_id = '0B_JDnoghFeEKLTlJT09IckMwOFk'
```

Comment:
Global variables should be in ALL_SNAKE_CAPS by PEP8 convention… I hate it but it does make it clearer when something is a global variable.